docs/src/modules/components/PickersPlayground.tsx

```diff
@@ -41,6 +43,32 @@ import { isDatePickerView } from '@mui/x-date-pickers/internals/utils/date-utils
 // eslint-disable-next-line no-restricted-imports
 import { isTimeView } from '@mui/x-date-pickers/internals/utils/time-utils';
 
+const ComponentSection = styled('div')(({ theme }) => ({
+  padding: theme.spacing(4),
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(2),
+  overflowX: 'auto',
+  '& .MuiPickersLayout-root': {
+    borderRadius: 2,
+    border: '1px dashed',
+    borderColor:
+      theme.palette.mode === 'light' ? theme.palette.divider : alpha(theme.palette.grey[500], 0.1),
+    ...(theme.palette.mode === 'dark' && {
```

Comment:
I've extracted the `Box` components into `styled('div')` to avoid having too complex and long prop descriptions for layout components.

Reply:
Thank you! 🎉